src/Resources/assets/controllers/combobox_controller.js

```diff
@@ -2,12 +2,11 @@ import { Controller } from '@hotwired/stimulus';
 import TomSelect from 'tom-select';
 import 'tom-select/dist/css/tom-select.css';
 
-
 export default class extends Controller {
     static values = {
         url: String,
-        required: Boolean,
-        min: 2
+        min: 2,
```

Comment:
@marcwieland95 Stimulus assumes the name on the left and the type on the right. `2` isn't a type.

This should be:

```js
{
  min: {
    type: Number,
    default: 2
  }
}
```